controls/pcidss_ocp4.yml

```diff
@@ -1979,82 +1995,117 @@ controls:
   levels:
   - base
   notes: ''
-  status: pending
+  status: automated
   controls:
   - id: Req-10.4.1
     title: 10.4.1 Critical systems have the correct and consistent time.
     levels:
     - base
-    notes: ''
-    status: pending
-    rules: []
+    status: automated
+    rules:
+      - service_chronyd_or_ntpd_enabled
+      - chronyd_or_ntpd_specify_remote_servers
 
   - id: Req-10.4.2
     title: 10.4.2 Time data is protected.
+    description: >-
+      Time data is protected as it is part of the underlying OS that is obfuscated from the OpenShift user interfaces.
     levels:
     - base
-    notes: ''
-    status: pending
-    rules: []
+    status: inherently met
 
   - id: Req-10.4.3
     title: 10.4.3 Time settings are received from industry-accepted time sources.
     levels:
     - base
-    notes: ''
-    status: pending
-    rules: []
+    status: automated
+    rules:
+      - chronyd_or_ntpd_specify_remote_servers
+      - chronyd_or_ntpd_specify_multiple_server
```

Comment:
interesting... we don't have an OpenShift-specific setting for this and these come from `linux_os/guide`... So they won't be included in the final profile as they're not applicable nor usable by the `ocp4` nor `ocp4-node` products. I think we should remove them and mark this rule as not applicable, as this is applicable to RHCOS instead. At some point we should have some RHCOS profile for this

Reply:
Ok, I wasn't sure if they were usable in this profile. Fixed.